src/xdocs/checks/coding/multiplestringliterals.xml

```diff
@@ -32,39 +32,25 @@
             </tr>
             <tr>
               <td>allowedDuplicates</td>
-              <td>
-                Specify the maximum number of occurrences to allow without generating a
-                warning.
-              </td>
+              <td>Specify the maximum number of occurrences to allow without generating a warning.</td>
               <td><a href="../../property_types.html#int">int</a></td>
               <td><code>1</code></td>
               <td>3.5</td>
             </tr>
+            <tr>
+              <td>ignoreOccurrenceContext</td>
+              <td>Specify token type names where duplicate strings are ignored even if they don't match ignoredStringsRegexp. This allows you to exclude syntactical contexts like annotations or static initializers from the check.</td>
+              <td><a href="../../property_types.html#subset of tokens TokenTypes">subset of tokens TokenTypes</a></td>
```

Comment:
it is same problems as https://github.com/checkstyle/checkstyle/issues/13722
we need definition of validation type, it is not always basic property type.